frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -1431,6 +1432,42 @@ export async function addForeignDataSet(
   });
   return result;
 }
+
+export async function exploreRemoteDataset(
+  remoteUris: string[],
+  credentials?: { username: string; pass: string },
+): Promise<DatasourceConfiguration> {
+  const { dataSource, report } = await Request.sendJSONReceiveJSON("/api/datasets/exploreRemote", {
+    data: credentials
+      ? remoteUris.map((uri) => ({
+          remoteUri: uri,
+          user: credentials.username,
+          password: credentials.pass,
+        }))
+      : remoteUris.map((uri) => ({ remoteUri: uri })),
+  });
+  console.log(report);
+  if (report[1].indexOf("403 Forbidden") !== -1 || report[1].indexOf("401 Unauthorized") !== -1) {
```

Comment:
Ah, I don’t think I noticed the [1] when you asked about this. I think it would be better to check if this appears in any of the report lines.

Reply:
Alright! Will adapt this 👍

Reply:
could the array be joined with `\n` by the back-end? or is there a reason to not do this?

Reply:
I could also do that in the backend, no problem, if that is easier to deal with for the frontend

Reply:
Would also improve the printed log :)

Reply:
pushed that now on the base branch :+1:

Reply:
I lied 😅 for the Eploration Log it's a bit more of a hassle now to display everything (because html doesn't display mulitline strings) but I think its better this way �
<img width="1157" alt="Screenshot 2022-08-11 at 16 39 57" src="https://user-images.githubusercontent.com/46814136/184159720-892df065-9398-4393-9ba2-f23772d74491.png">
�

Reply:
Not sure I understand right, do you want me to undo the change, then? :thinking:

Reply:
@Dagobert42 you can put the multiline string into a `<pre> ... </pre>` component.

Reply:
> Not sure I understand right, do you want me to undo the change, then? 🤔

No no all good. I'm solving it in the frontend.